src/parser/Statement.ts

```diff
@@ -293,6 +293,27 @@ export class End implements Statement {
     }
 }
 
+export class Stop implements Statement {
+    constructor(
+        readonly tokens: {
+            stop: Token;
+        }
+    ) {}
+
+    accept<R>(visitor: Visitor<R>): BrsType {
+        //TODO implement this in the runtime. It should pause code execution until a `c` command is issued from the console
+        throw new Error("Not implemented");
+    }
+
+    get location() {
+        return {
```

Comment:
I suppose we could have gotten away with `return this.tokens.stop.location` but 🤷‍♂